cypher/cypher-docs/src/docs/dev/ql/administration/security/role-management-syntax.asciidoc

```diff
@@ -1,85 +0,0 @@
-.Role management command syntax
```

Comment:
neo4j-documentation/4.3/cypher/cypher-docs/target/docs/dev/ql/administration/security/administration-security-users-and-roles.adoc